packages/localize/README.md

```diff
@@ -224,15 +222,14 @@ lit-localize supports two output modes: _transform_ and _runtime_.
 
 The `@lit/localize` module exports the following functions:
 
-> Note that lit-localize relies on distinctive, annotated TypeScript type
-> signatures to identify calls to `msg` and other APIs during analysis of your
-> code. Casting a lit-localize function to a type that does not include its
-> annotation will prevent lit-localize from being able to extract and transform
-> templates from your application. For example, a cast like
-> `(msg as any)("Hello")` will not be identified. It is safe to re-assign
-> lit-localize functions or pass them as parameters, as long as the distinctive
-> type signature is preserved. If needed, you can reference each function's
-> distinctive type with e.g. `typeof msg`.
+> Note that lit-localize relies on distinctive, annotated type signatures to
+> identify calls to `msg` and other APIs during analysis of your code. Casting a
+> lit-localize function to a type that does not include its annotation will
+> prevent lit-localize from being able to extract and transform templates from
+> your application. For example, a cast like `(msg as any)("Hello")` will not be
+> identified. It is safe to re-assign lit-localize functions or pass them as
```

Comment:
I assume it's _not_ safe to reassign in JS?

Reply:
Actually most simple patterns like a re-export or re-assign work just fine, since TS will follow references the same way it does in a .ts file.